thirdweb-dev/js
# [RN] Deprecates magicWallet in favor of magicLink for consistency

## Problem solved

We used `magicWallet` in RN vs `magicLink` in React. We want to be consistent across our stack.

## Changes made

magicWallet will show as deprecated and redirect users to magicLink
Renames magicWallet to magicLink and creates a new magicWallet alias for backward compat

## How to test

- [x] Manual tests: step by step instructions on how to test
Try to add `magicWallet` as a supportedWallet, you should see a deprecation warning
Use magicLink instead
